chroma-agent/chroma_agent/device_plugins/linux_components/block_devices.py

```diff
@@ -134,33 +141,142 @@ def fetch_device_list():
                 cmap(as_device), cfilter(filter_device), list)
 
 
-def add_to_ndp(xs, ys):
-    for x in xs:
-        for y in ys:
-            ndp.add_normalized_device(x, y)
+def parse_lvm_uuids(dm_uuid):
+    """
+    :param dm_uuid: device mapper uuid string (combined vg and lv with lvm prefix) from device_scanner
+    :return: tuple of vg and lv UUIDs
+    """
+    lvm_pfix = 'LVM-'
```

Comment:
How reliable is parsing `DM_UUID` to get this info?

Reply:
seems to be reliable that uuids are concatenated always for lvm devices in this manner (prefix-vguuid/lvuuid) and always 24 characters each